docs/diagrams/SaleExecuteAddSequenceDiagram.puml

```diff
@@ -93,6 +93,10 @@ deactivate CommandResult
 AddCommand --> LogicManager : result
```

Comment:
I think for `MassSaleCommandUtil`, there should be a `:` even if it's a static class. Also, would it be better to use `opt` instead of `alt` since there's only 1 alternative?